### DISCUSSION TITLE
🆕 Organization Discussions Private Beta

### DISCUSSION THREAD

**Discussion by evi-liu** [community/community#13192] [Discussions] [answered]
### Organizations Discussions are here 🚀!

You can now enable Discussions on the organization level, which will enable a broader space to host company or organization wide announcements and conversations. 

![image](https://user-images.githubusercontent.com/8809849/159378845-d12e6919-70a8-4fff-ae6c-000e8e084a0b.png)

### Want to try it out? 🧪 
If you're interested in joining the Organization Discussions private beta, **please respond below with the organization you'd like to enable it on** and I'll flag you in! 

Once you're flagged in, you can enable Organization Discussions under Organization Settings -> Discussions. You'll select a source repository to host the discussions, and then you can visit the Discussions tab on the Organization 🙌.  

Note that this is a private beta, so there may be some bugs you encounter. If you do experience any bugs or unexpected behavior, please leave a comment below so our team can address it and improve the feature. 

_Limited to the first 5 organizations that respond_

**Comment by arpitBhalla** [comment]
https://github.com/react-native-elements/react-native-elements

**Reply by stephaniegiang** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Comment by v1ctorio** [comment]
I would love to have that feature in my organization
https://github.com/nosesisaid
@nosesisaid

**Reply by stephaniegiang** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Comment by bobeagan** [comment]
Please enable beta for https://github.com/synapsestudios

**Reply by bobeagan** [reply]
Also, wondering if there are similar plans to enhance the current Teams discussion implementation to have all of these features too?

**Reply by huydeptra** [reply]
> Also, wondering if there are similar plans to enhance the current Teams discussion implementation to have all of these features too?

**Reply by stephaniegiang** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Comment by TechStudent10** [comment]
@Kendall-Does-Coding-Websites, @KendallDoesCoding-extensions, @DeeApps-Inc

**Reply by Beckyscrypto** [reply]
Pull request

**Reply by ehabzacharias** [reply]
Android System Web Version 79.3945.116

**Reply by Wilfredrick** [reply]
Hai semua org,, saya ada banyak melihat kepada itu,, la saya sahkan bila dah sampai masa nanti

**Reply by stephaniegiang** [reply]
All your organizations have been flagged in! Please give organization discussions a try and leave feedback below!

**Reply by TechStudent10** [reply]
Oh sorry I missed your comment haha. Well that doesn't matter now I guess

**Reply by RahmaNiftaliyev** [reply]
organization discussion is git beta and good logic comp

**Comment by volkantash** [comment]
Biz de katılabilir miyiz? (Can we join too?) 🤗
@turkce-yazilim-konati (30~300 üye/members)

**Reply by stephaniegiang** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Reply by volkantash** [reply]
> Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

Her şey yolunda görünüyor. (Everything looks fine.) Sağ olunuz @stephaniegiang. (Thanks)

Tek repo seçilmesini beklemiyordum. (I wasn't expecting a single repo to be selected.) Dilediğimiz tüm tartışmaları burada görebilmeyi isterdik. (We wish we could see all the discussions we could wish for here.) Bunun için şöyle bir yapı ağı kurulabilir. (For this, a structure network can be established.)

- Organizasyon tartışmalarına dilediğimiz repolar eklenmeli. (Any repos should be added to the organization discussions.)
- Repo tartışmalarında kategorileri, ayarlar bölümünden organizasyon altında gösterilip gösterilmeyeceğini ayarlayabilmeliyiz. (In the repo discussions, we should be able to set whether to show the categories under the organization from the settings section.)
- Organizasyon tartışmaları kategorisi düzenlenebilir olmalı. Bu kategoriler, repo tartışmalarının kategorileri ile eşleştirilmeli. (The organization discussions category must be editable. These categories should be matched with the categories of the repo discussions.)

İşte benim düşlediğim bir kullanış. (Here is a use I imagined.)
![resim](https://user-images.githubusercontent.com/25375593/161360762-446807cf-4879-40c2-9f78-2464832824b9.png)

**Comment by JaredTho-FQ** [comment]
FloQast wants in! https://github.com/FloQast

**Reply by stephaniegiang** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Comment by ss-o** [comment]
:octocat: https://github.com/z-shell

> Additionally, I have other org. which could participate in beta releases for group development if such an option exists.

Thanks,

**Reply by evi-liu** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Comment by laymonage** [comment]
Nice! Could you please enable the flag on http://github.com/giscus? Thanks!

**Reply by evi-liu** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Comment by LebzoDev** [comment]
Could you enable beta for this one https://github.com/api-platform ?? Best regards

**Reply by evi-liu** [reply]
Your organization has been flagged in! Please give organization discussions a try and leave feedback below!

**Comment by dr-orlovsky** [comment]
Please enable for @LNP-BP

**Comment by Totktonada** [comment]
I would have a look on the feature in https://github.com/tarantool

However I would like to share feedback even prior participating: the first thing I would ask is to have a look on discussions over the whole organization, not only just one repository.

**Comment by vmyazin** [comment]
Please link to https://github.com/Indeecast-Team/ 😌

**Comment by Salvinkay** [comment]
Please flag me in

**Comment by jonaharagon** [comment]
If you enable more, we'd like it on https://github.com/privacyguides

Initial impressions from our community (https://github.com/privacyguides/privacyguides.org/discussions/986) is that this discussions layout on other organizations looks more user-friendly than the standard interface :)

**Comment by warrenbuckley** [comment]
Hiya 👋
If your doing more rounds of invites than can the lovely Umbraco Community get some love and be invited into the ORG wide discussions beta for `umbraco`

Thanks

**Comment by evi-liu** [comment, marked as answer]
Organization Discussions is now fully shipped and generally available 🚀 . Read the changelog and see how to get started here: https://github.blog/changelog/2022-04-12-organization-discussions/

**Reply by volkantash** [reply]
🤗👏

Selam (Hi). Söz ettiğim şu özellik hakkında ne düşünüyorsunuz? (What do you think about this feature I'm talking about?)
https://github.com/github/feedback/discussions/13192#discussioncomment-2489433

**Reply by volkantash** [reply]
Selam (hi) @evi-liu 
🕷 Kendiliğinden gizlenmiş bir yorumun örküne tıklandığında işlemiyor... (It doesn't work when clicking the link of a self-hidden comment...)

**Reply by RahmaNiftaliyev** [reply]
organizational discussion how can l make it public ?

**Comment by vinckr** [comment]
Hello @evi-liu ,

The feature works great so far thanks! 

I was wondering if it is possible to move discussions from a repository to the organization discussions? I currently can only search for repositories it seems. 
So i want to move the discussions from [our old /meta repository](https://github.com/ory/meta/discussions) to [the new org discussions](https://github.com/orgs/ory/discussions).

Its not a huge thing just wondering if this is possible at the moment or planned to be added down the line.

**Reply by evi-liu** [reply]
@vinckr You can set the source repository on the Org Discussions page as ory/meta, so then you wouldn't have to migrate anything over. You can read more in the docs here: https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization

**Reply by vinckr** [reply]
Thanks that seems like a good solution.

**Comment by rungrotArunon** [comment]
Thanks you

**Comment by Wilfredrick** [comment]
Why this?

Fred82

Pada Kha, 5 Mei 2022, 16:12 nothing ***@***.***> menulis:

**Reply by rungrotArunon** [reply]
There's no reason, maybe because he graduated high in both directions and was taught by direct experience in the real world.

**Comment by rungrotArunon** [comment]
Because it's all opinions from adults.

**Comment by Wilfredrick** [comment]
Apakah itu?

Fred82

Pada Kha, 21 Apr 2022, 16:12 Vincent ***@***.***> menulis:
